BrowserKit/Sources/ComponentLibrary/Headers/HeaderView.swift

```diff
@@ -178,6 +190,24 @@ public final class HeaderView: UIView, ThemeApplicable {
         titleLabel.text = title
     }
 
+    public func setupDetails(subtitle: String, title: String, icon: UIImage?, warningIcon: String?, theme: Theme) {
+        titleLabel.font = FXFontStyles.Regular.body.scaledFont()
```

Comment:
This should probably be in the title label setup, as it doesn't change, right?

Reply:
Yes, so body can be by default... will move it to the label setup.